scripts/wsrep_sst_mariabackup.sh

```diff
@@ -681,8 +682,8 @@ cleanup_at_exit()
 
     if [ $estatus -ne 0 ]; then
         wsrep_log_error "Removing $MAGIC_FILE file due to signal"
-        [ -f "$MAGIC_FILE" ] && rm -f "$MAGIC_FILE" || :
-        [ -f "$DONOR_MAGIC_FILE" ] && rm -f "$DONOR_MAGIC_FILE" || :
+        [ -f "$MAGIC_FILE" ] && rm -rf "$MAGIC_FILE" || :
+        [ -f "$DONOR_MAGIC_FILE" ] && rm -rf "$DONOR_MAGIC_FILE" || :
```

Comment:
why change to use rm -rf here? target file is a simple file not a directory to be removed recursively